sdk/spring/scripts/replace_artifact.py

```diff
@@ -18,6 +18,20 @@
         'pom_list': [
             'sdk/cosmos/azure-spring-data-cosmos-test/pom.xml'
         ]
+    },
+    'spring': {
+        'artifact_dict': {
+            'azure-cosmosdb-spring-boot-2-3-starter': 'azure-cosmosdb-spring-boot-2-2-starter'
+        },
+        'pom_list': [
+            'sdk/spring/azure-spring-boot-test-cosmosdb/pom.xml'
+        ],
+        # update version of dependency, key = [group id]:[artfactid], value = (old_version, new_version)
```

Comment:
`value = (old_version, new_version)` , 2 items in comment, but 3 item in the following items.